go.mod

```diff
@@ -5,12 +5,11 @@ go 1.13
 require (
 	github.com/GoogleContainerTools/kpt v0.25.0
 	github.com/golang/groupcache v0.0.0-20200121045136-8c9f03a8e57e // indirect
-	github.com/golang/protobuf v1.3.5 // indirect
-	github.com/google/go-cmp v0.4.0 // indirect
 	github.com/spf13/cobra v0.0.5
 	golang.org/x/net v0.0.0-20200324143707-d3edc9973b7e // indirect
 	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d // indirect
 	golang.org/x/sys v0.0.0-20200331124033-c3d80250170d // indirect
 	google.golang.org/appengine v1.6.5 // indirect
+	google.golang.org/protobuf v1.24.0 // indirect
```

Comment:
what caused these changes? IIUIC, it should be scoped into the new `autogen` directory only

Reply:
Hmm this was probably added before I put the autogen in a separate `go.mod`. Will remove